bot_python_sdk/store.py

```diff
@@ -1,76 +1,107 @@
 import json
 import os
+from bot_python_sdk.logger import Logger
 
 _actions_file_path = 'storage/actions.json'
 _last_triggered_file_path = 'storage/last_triggered.json'
 _qr_image_path = 'storage/qr.png'
 _configuration_file_path = 'storage/configuration.json'
 _bot_public_key = 'storage/public.pem'
 
+LOCATION = 'Store'
+
 
 class Store:
+    def __init__(self):
+        pass
+
     @staticmethod
     def set_actions(actions):
-        file = open(_actions_file_path, 'w')
-        file.write(json.dumps(actions))
-        file.close()
+        try:
+            with open(_actions_file_path, 'w') as file:
+                file.write(json.dumps(actions))
+        except IOError as io_error:
+            Logger.error(LOCATION, 'Unable to write in {0} path'.format(_actions_file_path))
+            raise io_error.message
 
     @staticmethod
     def get_actions():
         if not os.path.isfile(_actions_file_path):
             return []
-        file = open(_actions_file_path, 'r')
-        actions = json.loads(file.read())
-        file.close()
-        return actions
+        try:
+            with open(_actions_file_path, 'r') as file:
```

Comment:
regards to your point about the file variable overwriting the default class, here we could go for actions_file for example. Minor improvement though.

Reply:
variable name has been changed as mentioned.